src/auth/use-subscribe-to-auth-user-change.ts

```diff
@@ -16,6 +16,7 @@ export const useSubscribeToAuthUserChange = (
     let signInInitiated = false;
     const unsubscribe = auth().onAuthStateChanged((user) => {
       if (user) {
+        Bugsnag.leaveBreadcrumb(`Auth state changed, sign-in type : ${mapAuthenticationType(user)}`);
```

Comment:
I think this will fire a little too often, as the `onAuthStateChanged` also fires when there has been no user change. Take a look at `SET_USER` in the reducer, where there is some logic for checking if it is a new user or not.

Reply:
@gorandalum I removed this and added the auth type as metadata to the breadcrumbs on the `SET_USER` reducer action, so it should no longer fire too often.